apps/web/screens/OfficialJournalOfIceland/OJOISearch.tsx

```
@@ -414,13 +511,74 @@ const OJOISearch: CustomScreen<OJOISearchProps> = ({
+    }
+  }
+
+  if (query.dagsTil && typeof query.dagsTil === 'string') {
+    const isValid = !isNaN(Date.parse(query.dagsTil))
+    if (isValid) {
+      dateTo = new Date(query.dagsTil).toISOString().split('T')[0]
+    }
+  }
+
+  if (query.sida && typeof query.sida === 'string') {
+    const check = !isNaN(parseInt(query.sida))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)